src/elections/services/ballotReady.service.ts

```diff
@@ -115,12 +115,28 @@ export class BallotReadyService {
 
   async fetchRacesByZipcode(
     zipcode: string,
+    level: string | null,
+    electionDate?: string | null,
     startCursor?: string | null,
   ): Promise<RacesByZipcode | null> {
-    const today = new Date().toISOString().split('T')[0]
-    const nextYear = new Date()
-    nextYear.setFullYear(nextYear.getFullYear() + 4)
-    const nextYearFormatted = nextYear.toISOString().split('T')[0]
+    let gt
+    let lt
+    if (electionDate) {
+      ;({ gt, lt } = getMonthBounds(electionDate))
+    } else {
+      gt = new Date().toISOString().split('T')[0]
+      const nextYear = new Date()
+      nextYear.setFullYear(nextYear.getFullYear() + 2)
+      lt = nextYear.toISOString().split('T')[0]
+    }
+
+    let levelWithTownship = level?.toUpperCase()
+    if (levelWithTownship === 'LOCAL') {
+      levelWithTownship = 'LOCAL,TOWNSHIP,CITY'
+    }
+    if (levelWithTownship === 'COUNTY') {
+      levelWithTownship = 'COUNTY,REGIONAL'
```

Comment:
Can we please reference the `const` we have exported here? 

https://github.com/thegoodparty/gp-api/blob/8e0369362cb9c48526b3366e7c09dca347770f63/src/shared/constants/governmentLevels.ts#L1